sktime/classification/deep_learning/cnn.py

```diff
@@ -59,6 +80,12 @@ def __init__(
         verbose=False,
         loss="mean_squared_error",
         metrics=None,
+        random_seed=None,
+        activation="sigmoid",
+        use_bias=True,
+        optimizer_=None,
```

Comment:
the init variables should have no underscore at the end.

What I meant is that these remain as you intended, without underscore.

You then copy a coerced/substituted version to the underscore version.

Reply:
I have made some changes, hope it satisfies. I am still unsure about what can be done for it, should I do a `deepcopy`?